programs/server/Server.cpp

```diff
@@ -1095,7 +1095,7 @@ int Server::main(const std::vector<std::string> & /*args*/)
             total_memory_tracker.setMetric(CurrentMetrics::MemoryTracking);
 
             auto * global_overcommit_tracker = global_context->getGlobalOvercommitTracker();
-            UInt64 max_overcommit_wait_time = config->getUInt64("global_memory_usage_overcommit_max_wait_microseconds", 200);
+            UInt64 max_overcommit_wait_time = config->getUInt64("global_memory_usage_overcommit_max_wait_microseconds", 5'000'000);
```

Comment:
Let's remove per-server global setting and rely only on the per-query setting `memory_usage_overcommit_max_wait_microseconds`.

By the way, you forgot to change it.